roles/edpm_nova_compute/defaults/main.yml

```diff
@@ -214,8 +215,8 @@ edpm_nova_compute_vendordata_dynamic_auth_user_domain_name: Default
 edpm_nova_compute_vif_plug_ovs_ovsdb_connection: ''
 edpm_nova_compute_vnc_enabled: true
 edpm_nova_compute_vncproxy_host: ''   # EndpointMap, NovaPublic, host_nobrackets
-edpm_nova_compute_vnc_server_listen: '{{ hostvars[inventory_hostname][service_net_map["nova_libvirt_network"] ~ "_ip"] }}'
-edpm_nova_compute_vncserver_proxyclient_address: '{{ hostvars[inventory_hostname][service_net_map["nova_libvirt_network"] ~ "_ip"] }}'
+edpm_nova_compute_vnc_server_listen: '{{ hostvars[inventory_hostname][edpm_service_net_map["nova_libvirt_network"] ~ "_ip"] }}'
+edpm_nova_compute_vncserver_proxyclient_address: '{{ hostvars[inventory_hostname][edpm_service_net_map["nova_libvirt_network"] ~ "_ip"] }}'
```

Comment:
i would not expect either of thise to use the libvirt network by defaut by the way.

they should actully be using the internal_api network.

the libvirt network is not nessisarly expected to be reacable by the novnc poxy and in fact for security reasons it reallly should not be.
when a seperate network is used for libvirt it should be isolated form the contolers and only accesable on the compute nodes.

again this is preexisitng behvior so not relvent to this review just noting that this is not what we woudl expect form a nova/compute perspective.